### PR TITLE
Fix: XMLHttpRequest failed to get header on TaoBao

### PR DESCRIPTION
**Pull request by yyycggg**
Re: #

### Changelog

* Fix: XMLHttpRequest failed to get header
* Fix: Modify the default value of Headers

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const _utils = require('../utils');
 
-const MAX_VALUE_WEBSOCkET = 1;
-let CURR_VALUE_WEBSOCKET = 0;
+const MAX_REF_WEBSOCKET = 1  // The maximum number of WEBSOCKET
```
Comment (minggo) [inline platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js]: Why change to ref?

Comment (yyycggg) [reply platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js]: The word `VALUE` is vague, Maybe the word `AMOUNT` is better than the word `REF`.

**Review by PPpro — APPROVED** [review]

File: platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js [hunk 2]
```diff
@@ -122,18 +122,18 @@ export default class WebSocket {
...
   }
 
-  _addRef(){
-    CURR_REF_WEBSOCKET += 1
+  _addCount(){
+    CURR_AMOUNT_WEBSOCKET += 1
   }
 
-  _reduceRef(){
+  _reduceCount(){
```
Comment (minggo) [inline platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js]: `add` and `reduce` are not antonyms. Maybe can use `increase` and `decrease`.

Comment (yyycggg) [reply platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js]: OK

**Review by minggo — APPROVED** [review]

**Pull request merged**